server/src/main/java/fr/uga/l3miage/tp1/exo2/models/UserEntity.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+
+@Table(name = "user_entity")
+@Entity
+public class UserEntity {
+
+    @Id
+    @Column(name = "idUser")
+    private Long idUser;
+
+    @Column(name = "SSO")
+    private boolean sso;
+
+    @Column(name = "lastConnexion")
+    private OffsetDateTime lastConnexion;
+
+    @OneToOne(mappedBy = "userEntityId")
+    private SessionEntity idSession;
+}
```

Comment:
idem, en mode `create` : 
```suggestion
@Entity
public class UserEntity {

    @Id
    private Long idUser;
    private boolean sso;
    private OffsetDateTime lastConnexion;

    @OneToOne(mappedBy = "userEntityId")
    private SessionEntity idSession;
}
```